cli/tsc/dts/lib.dom.d.ts

```diff
@@ -10909,6 +10909,7 @@ declare var PermissionStatus: {
 
 interface Permissions {
```

Comment:
Should this include revoke() and request()?

Reply:
@bartlomieju

Reply:
The changes to this file should be reverted - it's a file coming from TS and it describes the `DOM` library - these are browser permissions, not Deno permissions. The sync versions should only be added to `lib.deno.ns.d.ts`